components/collections/src/codepointinvlist/builder.rs

```diff
@@ -116,7 +116,7 @@ impl CodePointInversionListBuilder {
     /// let mut builder = CodePointInversionListBuilder::new();
     /// builder.add_u32(0x41);
     /// let check = builder.build();
-    /// assert_eq!(check.contains_u32(0x41), true);
+    /// assert_eq!(check.contains32(0x41), true);
     /// ```
     pub fn add_u32(&mut self, c: u32) {
```

Comment:
Is there a reason we didn't also change `add_u32` to `add32` here?

@robertbastian @dsipasseuth @echeran

Reply:
https://github.com/unicode-org/icu4x/issues/4354